tests/foreman/cli/test_computeresource_osp.py

```diff
@@ -16,19 +16,17 @@
 :Upstream: No
 """
 import pytest
+from box import Box
 from fauxfactory import gen_string
 
 from robottelo.cli.computeresource import ComputeResource
```

Comment:
is ComputeResource CLI still in use? why not to replace remaining usages with target_sat.cli?

Reply:
It is used in `cr_cleanup` function

Reply:
Can we remove this last usage of it and remove the import pls?

Reply:
Updated